aslam_offline_calibration/kalibr2/src/Image.cpp

```diff
@@ -0,0 +1,11 @@
+#include "kalibr2/Image.hpp"
+
+std::optional<aslam::cameras::GridCalibrationTargetObservation> ToObservation(const Image& image, const aslam::cameras::GridDetector& detector) {
+  auto observation = aslam::cameras::GridCalibrationTargetObservation(detector.target());
+  bool success = detector.findTargetNoTransformation(image.image, image.timestamp, observation);
```

Comment:
@frneer meta: this is heavily simplifying the kalibr implementation, isn't it? That's OK, but IMO we should document the code accordingly.